app/views/site/_study_resources_sidebar.html.erb

```diff
@@ -1,52 +1,50 @@
-<nav id="study-sidebar-content">
-  <%= link_to "<i class='fas fa-angle-double-right' id='resource-btn'></i>".html_safe, '#',
-              id: 'toggle-study-resources', class: 'btn btn-sm btn-default' %>
+<div id="study-sidebar-content">
   <% if @study.authors.corresponding.any? %>
-    <h3>Corresponding authors</h3>
+    <h5>Corresponding authors</h5>
     <% @study.authors.corresponding.each do |author| %>
-      <%= link_to "#{author.first_name} #{author.last_name} <i class='fa fas fa-envelope'></i>".html_safe, '#',
-                  class: 'btn btn-sm btn-default corresponding-author',
+      <%= link_to '#',
+                  class: 'corresponding-author',
                   data: {
                     contact: author.base_64_email, placement: 'top',
                     'analytics-name' => 'email-corresponding-author'
                   },
-                  title: "Send email to #{author.first_name} #{author.last_name}" %>
+                  title: "Send email to #{author.first_name} #{author.last_name}" do %>
+        <%= author.first_name %> <%= author.last_name %> <i class='fa fas fa-envelope'></i>
+      <% end %>
+      <br/>
     <% end %>
   <% end %>
   <% if @study.publications.any? %>
-    <h3>Related publications</h3>
+    <h5>Related publications</h5>
     <% @study.publications.each do |publication| %>
-      <h4><%= link_to publication.title, publication.url, target: :_blank, rel: 'noopener',
+      <%= link_to publication.url, target: :_blank, rel: 'noopener',
                       'data-analytics-name' => 'publication-link-journal', title: "View in #{publication.journal}",
-                      data: { toggle: 'tooltip' } %>
-      </h4>
+                      data: { toggle: 'tooltip' } do %>
+        <span class="underline"><%= publication.title %></span> <span class="fas fa-external-link-alt"></span>
```

Comment:
I see this in the [Terra style guide](https://www.sketch.com/s/cca3a834-985b-4165-bbae-21f615aebd2d/a/8d20QD), and so I don't insist on changing it, but I think it's worth noting this is pretty noisy.  

I strongly suspect this is not required for accessibility compliance, as e.g. https://www.nih.gov/ uses the more common modern practice of not underlining links.

Reply:
this underlining is based on the feedback from demo that it was hard to see that the publication link was also a header for the other information underneath it (pubmed link, citation, etc...).  Agree that it is not necessary for accessibility, it was just for visual clarity